app/Models/V2/Projects/Project.php

```diff
@@ -496,4 +498,9 @@ private function submittedSiteReportIds(): array
     {
         return $this->submittedSiteReports()->pluck('v2_site_reports.id')->toArray();
     }
-}
+
+    public function getTotalSitePolygons()
+    {
+        return $this->sitePolygons()->count();
+    }
```

Comment:
This would be more idiomatic as an attribute:
```
public function getTotalSitePolygonsAttribute()
{
   return $this->sitePolygons()->count();
}
```

Then in the controller on line 85, you'd reference like this instead:
```
$countSitePolygons = $project->total_site_polygons;
```
Or just skip assigning it to a separate variable entirely and reference the attribute directly on line 96.

Finally, the `?? 0` on line 85 and `?? null` on line 96 in the controller and not needed. A `count()` call automatically returns 0 if there are no associated records.

Reply:
removed and used directly the attribute accesor.